tools/docker

```diff
@@ -21,7 +21,7 @@ ROOT=$(dirname "$WD")
 set -eu
 
 # LOCAL_OUT is set when runing in a Makefile and set to current architecture (not host)
-if [[ ! -z "${LOCAL_OUT}" ]]; then
+if [[ ! -z "${LOCAL_OUT+.}" ]]; then
```

Comment:
Won't this always return true? "$LOCAL_OUT+." should always be a string do -z should return false, and then we make that true?